server/etcdserver/api/etcdhttp/metrics.go

```diff
@@ -65,7 +67,15 @@ func NewHealthHandler(lg *zap.Logger, hfunc func(excludedAlarms AlarmSet) Health
 			return
 		}
 		excludedAlarms := getExcludedAlarms(r)
-		h := hfunc(excludedAlarms)
+		// Kubernetes Probes (i.e. livenessProbe) use "/health" endpoint to make a decision whether to restart a specific container.
```

Comment:
minor nit: Can we condense the docs a bit, this feels overly verbose. Perhaps it would be good for folks revisiting this PR to add the full docs to the PR description above. Perhaps something like below?

```
Passing the query parameter "serializable=true" ensures that the health of the local etcd is checked vs the health of the cluster. This is useful for probes attempting to validate the liveness of the etcd process vs readiness of the cluster to serve requests. 
```

Reply:
@hexfusion  Thanks for the comment. Fixed. 

I also squashed all the commits.